packages/cxl-lumo-styles/scss/themes/vaadin-accordion.scss

```diff
@@ -3,3 +3,8 @@
   column-rule: 1px solid var(--lumo-shade-10pct); // match panel `outline`
   column-width: calc(var(--cxl-wrap-width) / 4);
 }
+
+
+:host([theme~="cxl-hub-sidebar"]) {
+  // background: pink;
+}
```

Comment:
Please avoid completely empty commented out blocks. This brings in files into review that don't need to show up at all, increasing work.

Reply:
sorry, pinky debug, gone